keras/backend/jax/random.py

```diff
@@ -81,6 +82,24 @@ def dropout(inputs, rate, noise_shape=None, seed=None):
     )
 
 
+def alpha_dropout(inputs, rate, noise_shape=None, seed=None):
+    noise_shape = _get_concrete_noise_shape(inputs, noise_shape)
+    alpha = 1.6732632423543772848170429916717
```

Comment:
Any specific reason for having more than 6-7 decimal points?

Reply:
In practice it will be cast to float32 so some precision will be lost. The number above is just taken from the paper.